x/auth/types/account_test.go

```diff
@@ -214,6 +215,28 @@ func TestModuleAccountJSON(t *testing.T) {
 	require.Equal(t, acc.String(), a.String())
 }
 
+func TestModuleAccountJSONPB(t *testing.T) {
+	pubkey := secp256k1.GenPrivKey().PubKey()
+	addr := sdk.BytesToAccAddress(pubkey.Address())
+	baseAcc := types.NewBaseAccount(addr, nil, 10, 50)
+	acc := types.NewModuleAccount(baseAcc, "test", "burner")
+
+	jm := jsonpb.Marshaler{}
+	bz, err := acc.MarshalJSONPB(&jm)
+	require.NoError(t, err)
+
+	jum := jsonpb.Unmarshaler{}
+	addr2 := sdk.AccAddress("")
+	baseAcc2 := types.NewBaseAccount(addr2, nil, 0, 0)
+	acc2 := types.NewModuleAccount(baseAcc2, "")
+	err = acc2.UnmarshalJSONPB(&jum, bz)
+	require.NoError(t, err)
+
+	// error on bad bytes
+	err = acc2.UnmarshalJSONPB(&jum, bz[:len(bz)/2])
+	require.NotNil(t, err)
```

Comment:
```suggestion
	require.Error(t, err)
```
How about using `Error()`?